modules/i3s/src/lib/parsers/parse-slpk/index.ts

```diff
@@ -0,0 +1,57 @@
+import {SlpkLoaderOptions} from 'modules/i3s/src/i3s-slpk-loader';
```

Comment:
In loaders.gl we never use `index.ts` files inside modules. when dealing with a lot of files, the extra nesting confuses more than it helps.

Rename this file to `parse-slpk.ts`?